### PR TITLE
fix(aio): fix sidenav and hamburger button alignment

### PR DESCRIPTION
**Pull request by sjtrimble**
- Fix sidenav height to align with topnav toolbar resizing at 600px
- Fix positioning of sidenav to match up with topnav toolbar resizing at 600px

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by IgorMinar — COMMENTED** [review]
I can't tell what this change is doing. The PR preview doesn't seem to be any different from what's on the staging site.

**Comment by petebacondarwin** [comment]
Neither can I. Can you provide screenshots @sjtrimble ?

**Comment by sjtrimble** [comment]
Here is what I was seeing - example from the staging site:

sidenav-mobile
sidenav

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 1]
```diff
@@ -32,10 +32,18 @@ md-sidenav.mat-sidenav.sidenav {
   min-width: 260px;
   background-color: $offwhite;
   box-shadow: 6px 0 6px rgba(0,0,0,0.10);
-  height: calc(100vh - 64px);
+  height: calc(100vh - 66px);
+
+  @media (max-width: 600px) {
+    height: calc(100vh - 56px);
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_sidenav.scss]: Shouldn't we also change the `top` to `56px`?

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 2]
```diff
@@ -32,10 +32,18 @@ md-sidenav.mat-sidenav.sidenav {
...
+  }
 
   &.collapsed {
-    top: 56px;
+    top: 66px;
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_sidenav.scss]: Shouldn't this be `64px`?

File: aio/src/styles/1-layouts/_sidenav.scss [hunk 3]
```diff
@@ -32,10 +32,18 @@ md-sidenav.mat-sidenav.sidenav {
   min-width: 260px;
   background-color: $offwhite;
   box-shadow: 6px 0 6px rgba(0,0,0,0.10);
-  height: calc(100vh - 64px);
+  height: calc(100vh - 66px);
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_sidenav.scss]: Why `66px`?

**Comment by petebacondarwin** [comment]
Some questions to answer @sjtrimble

**Comment by petebacondarwin** [comment]
Closing due to lack of activity and at least the second screenshot appears to be fixed on staging.

**Comment by sjtrimble** [comment]
@petebacondarwin I know this was closed a bit ago -- but I'm still seeing the issue (see image) where the hamburger "square" covers part of the white area of the sidenav drawer. 

hamburger-x

**Comment by petebacondarwin** [comment]
@sjtrimble if you have time to answer @gkalpak's questions then we could reopen and land this PR.

**Pull request closed without merging**